hackathon_site/event/jinja2/event/form_base.html

```diff
@@ -0,0 +1,25 @@
+{% extends "event/base.html" %}
+
+{% block body %}
+<div class="container">
+    <div class="section">
+        <div class="loginDiv z-depth-3 row">
+            <div class="row col s12 flexColCenter {% if self.form_img() %}l7 m8{% endif %}">
+                <h1 class="formH1">{% block form_title %}{% endblock %}</h1>
+                <form class="col s12">
+                    {% block form_inputs %}{% endblock %}
+                </form>
+
+                <a href="#" class="btn-small waves-effect waves-light secondaryColor colorBtn center">{% block form_button %}{% endblock %}</a>
```

Comment:
Each form is going to need a way to alter what this button does, probably by making it an actual button with an `onClick`. Having the classes in the base template is nice, but I think the `form_button` block needs to capture everything, and each instance will have to apply the classes. Any workarounds?